src/metabase/models.clj

```diff
@@ -138,3 +143,33 @@
  [timeline-event TimelineEvent]
  [user User]
  [view-log ViewLog])
+
+(defenterprise resolve-enterprise-model
+  "TODO"
+  metabase-enterprise.models
+  [x]
+  x)
+
+(methodical/defmethod t2.model/resolve-model :before :default
```

Comment:
I moved these here since otherwise there's a circular dependency due to the inclusion of `metabase.public-settings.premium-features` in `metabase.models.interface`